integration-test/src/test/resources/com/sonar/it/java/PmdTest/pmd-all-rules.xml

```diff
@@ -298,12 +298,6 @@
       <repositoryKey>pmd7</repositoryKey>
```

Comment:
ik denk dat deze key gaat zorgen voor een gehele rule definitie re-fresh! Want Kotlin regels konden zo naast java regels bestaan.

Reply:
should we use a separate key for kotlin? e.g. `pmd-kotlin`? To avoid same rule name clashes.